dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Current/PhaseShifterBlockingI.mo

```diff
@@ -0,0 +1,46 @@
+within Dynawo.Electrical.Controls.Current;
+/*
+* Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+model PhaseShifterBlockingI "Phase Shifter blocking model"
+
+  import Modelica.Constants;
+  import Dynawo.Connectors;
+  import Dynawo.NonElectrical.Logs.Timeline;
+  import Dynawo.NonElectrical.Logs.TimelineKeys;
+  import Dynawo.Types;
+
+  public
+    parameter Types.CurrentModule IMax "Maximum current on the monitored component";
+    parameter Types.Time tLagBeforeActing "Time lag before taking action";
+    Types.CurrentModule IMonitored "Monitored current";
```

Comment:
Could you separate the parameters and the variables with a blank line ?

Reply:
done, thx